src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,57 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    private static final int MAX_SIZE = 12;
+    private static final int EMPTY = -1;
+    private static final int FULL = 11;
+    private final int[] numbers = new int[MAX_SIZE];
+
+    private int total = EMPTY;
+
+    public static int getMaxSize() {
+        return MAX_SIZE;
+    }
+
+    public static int getEMPTY() {
+        return EMPTY;
+    }
+
+    public static int getFULL() {
+        return FULL;
+    }
```

Comment:
Task 10 in "accessory i hermetyzacja" concerned only 'total' getter.

Task 11 in "accessory i hermetyzacja" concerned only non-private fields.